verbose_debugging_reports.md

```diff
@@ -33,7 +33,7 @@ The `body` will contain the following fields for all source debugging reports:
   This will be a string if there was one such site, or a list of strings if
   there were multiple.
 * `source_event_id`: The source registration's `source_event_id`.
-* `source_site`: The top-level site on which the source registration occurred.
+* `source_site`: The top-level site, or the embedded site for specific use cases, on which the source registration occurred.
```

Comment:
add (for example, [AMP](https://amp.dev) pages rendered by an AMP
cache/viewer provider) here as well